django/contrib/postgres/search.py

```diff
@@ -92,14 +92,38 @@ def __init__(self, lhs, connector, rhs, config, output_field=None):
         super(CombinedSearchVector, self).__init__(lhs, connector, rhs, output_field)
 
 
-class SearchQuery(Value):
+class SearchQueryCombinable(object):
+    BITAND = '&&'
+    BITOR = '||'
+
+    def _combine(self, other, connector, reversed, node=None):
+        if not isinstance(other, SearchQueryCombinable) or not self.config == other.config:
```

Comment:
I don't see any tests failing with these first 4 lines removed. Could you test them or remove them if they're unneeded?